pkg/accounting/projectusage_test.go

```diff
@@ -30,69 +30,118 @@ func TestProjectUsage(t *testing.T) {
 	}{
 		{name: "doesn't exceed storage or bandwidth project limit", expectedExceeded: false, expectedErrMsg: ""},
 		{name: "exceeds storage project limit", expectedExceeded: true, expectedResource: "storage", expectedErrMsg: "segment error: metainfo error: rpc error: code = ResourceExhausted desc = Exceeded Alpha Usage Limit; segment error: metainfo error: rpc error: code = ResourceExhausted desc = Exceeded Alpha Usage Limit"},
-		{name: "exceeds bandwidth project limit", expectedExceeded: true, expectedResource: "bandwidth", expectedErrMsg: "segment error: metainfo error: rpc error: code = ResourceExhausted desc = Exceeded Alpha Usage Limit; segment error: metainfo error: rpc error: code = ResourceExhausted desc = Exceeded Alpha Usage Limit"},
 	}
 
 	testplanet.Run(t, testplanet.Config{
 		SatelliteCount: 1, StorageNodeCount: 6, UplinkCount: 1,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
 		saDB := planet.Satellites[0].DB
-		orderDB := saDB.Orders()
 		acctDB := saDB.Accounting()
 
-		// Setup: This date represents the past 30 days so that we can check
-		// if the alpha max usage has been exceeded in the past month
-		from := time.Now().AddDate(0, 0, -accounting.AverageDaysInMonth)
+		// Setup: create a new project to use the projectID
+		projects, err := planet.Satellites[0].DB.Console().Projects().GetAll(ctx)
+		projectID := projects[0].ID
+		require.NoError(t, err)
 
 		for _, tt := range cases {
 			t.Run(tt.name, func(t *testing.T) {
 
-				// Setup: create a new project to use the projectID
-				projects, err := planet.Satellites[0].DB.Console().Projects().GetAll(ctx)
-				projectID := projects[0].ID
-				require.NoError(t, err)
-				bucketName := "testbucket"
-				bucketID := createBucketID(projectID, []byte(bucketName))
-
 				// Setup: create BucketStorageTally records to test exceeding storage project limit
 				if tt.expectedResource == "storage" {
 					now := time.Now()
 					err := setUpCreateTallies(ctx, projectID, acctDB, now)
 					require.NoError(t, err)
 				}
 
+				// Execute test: get storage totals for a project, then check if that exceeds the max usage limit
+				inlineTotal, remoteTotal, err := acctDB.ProjectStorageTotals(ctx, projectID)
+				require.NoError(t, err)
+				maxAlphaUsage := 25 * memory.GB
```

Comment:
Why using `accounting.ExpansionFactor` in bandwidth test and not here?

Reply:
oops let me fix that